fix typos

RainRat opened the pull request.

RainRat committed:
fix typos

RainRat committed:
fix typos

joshkurien commented:
xD
oh my goodness, didn't notice all this until now.
I think translation isn't done by PRs though

RainRat commented:
Yeah, it's been pointed out. I'll save my work here until I'm ready to upload.

RainRat committed:
interestingly, 'everytime' and 'noone' aren't actual words

RainRat committed:
more typo fixes

RainRat committed:
fix typos

RainRat committed:
Merge branch 'GreyWolfDev:beta' into beta

RainRat committed:
fix typos

RainRat committed:
Merge branch 'GreyWolfDev:beta' into beta

RainRat committed:
Merge branch 'GreyWolfDev:beta' into beta

RainRat committed:
Merge branch 'GreyWolfDev:beta' into beta

RainRat committed:
Merge branch 'GreyWolfDev:beta' into beta

RainRat committed:
fix typos

RainRat committed:
Fix spelling errors across config, code, and UI

.gitignore:
- correct 'Interal.settings' → 'Internal.settings'
- update comment: 'ignoreable' → 'ignorable'

Code:
- rename 'Maintenenace' → 'Maintenance' method
- rename 'NewNodeThreshhold' → 'NewNodeThreshold' and update references

Comments & UI:
- fix 'librari' → 'library' in Updater.cs comment
- correct 'asronist' → 'arsonist' in Index.cshtml

RainRat committed:
Merge branch 'GreyWolfDev:beta' into beta

RainRat committed:
Merge branch 'GreyWolfDev:beta' into beta

RainRat committed:
Merge branch 'GreyWolfDev:beta' into beta
